sentinel5dl/__main__.py

```diff
@@ -141,8 +142,22 @@ def main():
             Example: 2019-09-17T23:59:59.999Z'''
     )
 
+    parser.add_argument(
+        '--use-certifi',
+        default='false',
+        action='store_true',
+        help='''If Certificate Authority (CA) bundle is not already supplied
+            by your operating system, certifi provides an easy way of
+            providing a cabundle.'''
+    )
+
     args = parser.parse_args()
 
+    # Provide a Certificate Authority (CA) bundle
+    if args.use_certifi is True:
```

Comment:
This is only needed because of the string `"false"` being the default. Applying the other suggestion, the specific comparison to `True` can go.
```suggestion
    if args.use_certifi:
```
Btb, necessary comparisons to `True` or `False` are extremely rare, thus whenever you encounter this it's likely a sign for something being weird 😸